docs/release_notes.rst

```diff
@@ -5,6 +5,21 @@ v4.3.2
 ----------
 
 * Fix discrepancy between runtime and type-checker's perspective of Index and derived types
+* Add ListAttribute.remove_indexes action for removing specific indexes from a ListAttribute
+* Type stub fixes:
+  * Add missing parameters of Model.scan
+  * Change hash_key parameter to be typed Any
```

Comment:
```suggestion
    * Add missing parameters of Model.scan
    * Change hash_key parameter to be typed Any
```

Otherwise this doesn't render quite right

![image](https://user-images.githubusercontent.com/252209/80048238-98895f80-84dd-11ea-9a15-a62dbf4bbbb4.png)